.github/workflows/ci.yml

```
@@ -181,6 +181,9 @@ jobs:
           name: babel-artifact
       - name: Lint
         run: make -j tscheck lint-ci
+      - name: Ensure cwd does not contain uncommitted changes
+        run: |
+          node ./scripts/assert-dir-git-clean.js lint
```

```suggestion
          node ./scripts/assert-dir-git-clean.js code-quality
```

`lint` does not trigger `tscheck`.